### PR TITLE
Fix schema and reached 100% coverage in schema

### PR DESCRIPTION
**Pull request by berenda-andrey**

### CODE REVIEW

File: winter/schema/method_arguments_inspector.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import abc
+from typing import List
+
+from drf_yasg import openapi
+
+from ..core import ComponentMethod
+
+
+class MethodArgumentsInspector(abc.ABC):
```
Comment (berenda-andrey) [inline winter/schema/method_arguments_inspector.py]: It just renamed from ControllerMethodInspector

Comment (berenda-andrey) [reply winter/schema/method_arguments_inspector.py]: not removed and added

File: tests/test_schema/test_inspectors.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    assert valid_parameter.type == openapi.TYPE_INTEGER
+    assert valid_parameter.description == 'Valid doc'
+
+    invalid_parameter = parameter_by_name['invalid_param']
+    assert invalid_parameter.type == openapi.TYPE_STRING
+    assert invalid_parameter.description == 'Invalid doc (Note: parameter type can be wrong)'
+    assert 'not_in_method' not in parameter_by_name
+
+
+def test_query_parameter_inspector():
```
Comment (mofr) [inline tests/test_schema/test_inspectors.py]: Please split path and query inspectors tests into two files.

File: tests/test_schema/test_inspectors.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
+from drf_yasg import openapi
+
```
Comment (mofr) [inline tests/test_schema/test_inspectors.py]: Move this file to `tests/schema`

File: winter/pagination/page_position_argument_inspector.py [hunk 4]
```diff
@@ -5,10 +5,10 @@
 from .page_position import PagePosition
 from .page_position_argument_resolver import PagePositionArgumentResolver
 from ..core import ComponentMethod
-from ..schema import ControllerMethodInspector
+from ..schema import MethodArgumentsInspector
 
 
-class PagePositionArgumentInspector(ControllerMethodInspector):
+class PagePositionArgumentInspector(MethodArgumentsInspector):
```
Comment (mofr) [inline winter/pagination/page_position_argument_inspector.py]: MethodArgumentsInspector -> MethodArgumentInspector
or
PagePositionArgumentInspector -> PagePositionArgumentsInspector

File: winter/schema/generation.py [hunk 5]
```diff
@@ -5,7 +5,7 @@
 from drf_yasg import openapi
 from drf_yasg.utils import swagger_auto_schema
 
-from .controller_method_inspector import get_controller_method_inspectors
+from .method_arguments_inspector import get_controller_method_inspectors
```
Comment (mofr) [inline winter/schema/generation.py]: get_controller_method_inspectors -> get_method_arguments_inspectors

File: winter/schema/generation.py [hunk 6]
```diff
@@ -40,32 +41,32 @@ def generate_swagger_for_operation(view_func, controller, route: Route):
...
         return output_serializer.class_(**output_serializer.kwargs)
 
     type_hints = typing.get_type_hints(method.func)
     return_value_type = type_hints.get('return', None)
-    type_info = inspect_type(return_value_type)
-    return type_info.get_openapi_schema()
+    try:
+        type_info = inspect_type(return_value_type)
+    except InspectorNotFound:
+        return default
```
Comment (mofr) [inline winter/schema/generation.py]: Maybe, return None in this case?

**Pull request merged**